src/game/phase/dungeon.cpp

```diff
@@ -72,7 +86,7 @@ proto::Phase DungeonPhase::s_update() {
     // Send the position and orientation of the specified game objects.
     // Currently sending all Player objects and Constructs.
     for (auto const & o : GameObject::game_objects) {
-        if (dynamic_cast<Player*>(o.second) || dynamic_cast<Construct*>(o.second))
+        if (dynamic_cast<Player*>(o.second) || dynamic_cast<Construct*>(o.second) || dynamic_cast<Essence*>(o.second))
```

Comment:
This might get pretty fat later on, maybe have some variable in Gameobject that indicates whether or not it needs updating and check for that instead. Each subclass of GameObject can then set it according to its own needs.